catalog/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,168 @@
+local id = 'projects/edf-methanesat-ee/assets/public-preview/L3concentration';
+local subdir = 'edf-methanesat-ee';
+local version = '1.0.0';
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+local license = spdx.proprietary;
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  title: 'MethaneSAT L3 Concentration Public Preview V' + version,
+  version: version,
+  description: |||
+    This early "Public Preview" dataset provides geospatial data for the 
+    column-averaged dry-air mole fraction of methane in the atmosphere, "XCH₄", 
+    retrieved from measurements by the MethaneSAT imaging spectrometer. XCH₄ is 
+    defined as the total column amount (number of molecules above a unit surface 
+    area) of methane ("CH₄") divided by the total amount of air (number of molecules 
+    above a unit surface area, water vapor amount removed) along the line of sight 
+    from the satellite’s spectrometer to the Earth's surface and back up to the Sun. 
+    Additional data layers are provided for reference: observed albedo at 1622 nm, 
+    surface pressure, and terrain height.
+    The spatial footprint of individual MethaneSAT soundings is 110m x 400m when 
```

Comment:
please add a new line to start a new paragraph

Reply:
done